compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -2043,6 +2045,10 @@ impl<'tcx> TyCtxt<'tcx> {
         self.typeck(self.hir().body_owner_def_id(body))
     }
 
+    pub fn output_filenames(self) -> &'tcx Arc<OutputFilenames> {
+        self.output_filenames_raw(())
+    }
```

Comment:
Nit: As `output_filenames_raw` cannot return an `Err` any more, do we need to keep the `output_filenames_raw`/`output_filenames` distinction?

Reply:
It's mostly a small annoyance that we have to call the query as `output_filenames(())`

Reply:
Considering that's what we had before this PR, I'll revert